src/custom/state/claim/hooks/index.ts

```diff
@@ -1005,124 +1000,3 @@ export const useClaimLinks = () => {
     [chainId]
   )
 }
-
-/**
- * Hook that parses the result input with BigNumber value to CurrencyAmount
- */
-function useParseVCowResult(result: CallStateResult | undefined) {
```

Comment:
Any reason to remove just part of claim hook methods and not all of them?

Reply:
This part of code was already moved to `src/custom/state/cowToken` but this might be just some bad merge. We also want to keep claim related code in there.